pypeit/deprecated/script_utils.py

```diff
@@ -0,0 +1,319 @@
+"""
```

Comment:
I think this is touched in another open PR by @rcooke-ast

Reply:
Hah - I just deleted the file... @kbwestfall has appropriately moved this to deprecated, which is the right thing to do.